packages/frontend/src/components/UserSettings/ProfilePanel/index.tsx

```diff
@@ -112,116 +110,101 @@ const ProfilePanel: FC = () => {
-                            <Tooltip2 content="This e-mail has been verified">
-                                <Box p={6}>
-                                    <Icon
-                                        icon="tick-circle"
-                                        color={Colors.GREEN4}
-                                    />
-                                </Box>
-                            </Tooltip2>
-                        ) : isEmailServerConfigured && !data?.isVerified ? (
-                            <Box p={6}>
-                                <Icon icon="issue" color={Colors.GRAY3} />
-                            </Box>
-                        ) : undefined
+                            <Tooltip
+                                label="This e-mail has been verified"
+                                withArrow
+                            >
+                                <MantineIcon
+                                    size="lg"
+                                    icon={IconCircleCheck}
+                                    color="green.6"
+                                />
+                            </Tooltip>
+                        ) : (
+                            <MantineIcon
+                                size="lg"
+                                icon={IconAlertCircle}
+                                color="gray.6"
+                            />
+                        )
                     }
                 />
-                {isEmailServerConfigured && !data?.isVerified ? (
-                    <Text color="dimmed" mt="sm">
-                        This email has not been verified.{' '}
-                        <Anchor
-                            component="span"
-                            onClick={() => {
-                                if (!data?.otp) {
-                                    sendVerificationEmail();
-                                }
-                                setShowVerifyEmailModal(true);
-                            }}
-                        >
-                            Click here to verify it
-                        </Anchor>
-                        .
-                    </Text>
-                ) : null}
-            </FormGroup>
+            </Stack>
 
-            <div style={{ flex: 1 }} />
+            {isEmailServerConfigured && !data?.isVerified ? (
+                <Text color="dimmed" mt="sm">
+                    This email has not been verified.{' '}
+                    <Anchor
+                        component="span"
+                        onClick={() => {
+                            if (!data?.otp) {
+                                sendVerificationEmail();
+                            }
+                            setShowVerifyEmailModal(true);
+                        }}
+                    >
+                        Click here to verify it
+                    </Anchor>
+                    .
+                </Text>
+            ) : null}
```

Comment:
can you try using this? description labels and you can change order if necesary https://mantine.dev/core/input/#inputwrapperorder

Reply:
that's great! 

looks like this now: 
![image](https://user-images.githubusercontent.com/7611706/235909374-61a949aa-4d5b-4262-a439-26008188065d.png)

thanks